packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from '@mongosh/shell-api';
 
 interface Container {
```

Comment:
Just noticed `Container` here sounds super generic, this one maybe deserves a more clarifying name or at least a comment. That one could actually even be `Config`. Would be appropriate as a contract to redefine it here as the subset of the config that needs to be accessed from ShellEvaluator.

But yeah, up to you if you want to do anything about it.

Reply:
I thought about it, and I think it makes sense to just turn `enableTelemetry()` and `disableTelemetry()` into actual functions that then call a method on the `EvaluationListener`, just like we do for `console.log()` etc.